src/lambdas/initiateDataRequest/isEmailInValidRecipientList.ts

```diff
@@ -0,0 +1,16 @@
+import { readS3DataToString } from '../../sharedServices/s3/readS3DataToString'
+import { getEnv } from '../../utils/helpers'
+export const isEmailInValidRecipientList = async (
+  recipientEmail: string
+): Promise<boolean> => {
+  console.log(
+    `Checking if recipient email ${recipientEmail} is in the pre-defined list of recipients`
+  )
+
+  const validRecipientList = await readS3DataToString(
+    getEnv('VALID_EMAIL_RECIPIENTS_BUCKET'),
+    'valid-email-recipients.txt'
+  )
+  console.log('this is the list of recipients: ', validRecipientList)
```

Comment:
Not sure we want to log all the emails on every request.  Once we've tested this could we update this to just say whether the validRecipientList has a length greater than 0?

Reply:
Yep, definitely, to be honest this was actually just debug code!

Reply:
I've updated this now